app/src/main/java/com/monkeyteam/chimpagne/MainActivity.kt

```diff
@@ -133,15 +142,34 @@ class MainActivity : ComponentActivity() {
               myEventsViewModel.fetchMyEvents()
               MyEventsScreen(navObject = navActions, myEventsViewModel = myEventsViewModel)
             }
-            composable(Route.VIEW_DETAIL_EVENT_SCREEN + "/{EventID}") { backStackEntry ->
-              ViewDetailEventScreen(
-                  navObject = navActions,
-                  eventViewModel =
-                      viewModel(
-                          factory =
-                              EventViewModel.EventViewModelFactory(
-                                  backStackEntry.arguments?.getString("EventID"), database)))
-            }
+            composable(
+                route = Route.VIEW_DETAIL_EVENT_SCREEN + "/{EventID}",
+                deepLinks =
+                    listOf(
+                        navDeepLink {
+                          uriPattern = "https://www.manigo.ch/events/?uid={EventID}"
```

Comment:
A constant should be declared somewhere with the domain used for urls, and used here